### PR TITLE
fix(frontend): fix display of progress bar in RTL languages

### PR DESCRIPTION
**Pull request by raphj**
#5326 

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link) [I didn't write tests, seems hard to do and unlikely to regress)
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [X] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/ProgressBar/ProgressBar.styl [hunk 1]
```diff
@@ -8,6 +8,18 @@
   margin 0 auto
   position relative
 
+// RJ - 2022-02-01 - NOTE
+// the right ⟷ left changes in [dir=rtl] blocks below fix the broken appearance
+// of the progress bar for RTL langages, but make the progress go from
```
Comment (raphj) [inline frontend_lib/src/component/ProgressBar/ProgressBar.styl:13]:
```suggestion
// of the progress bar for RTL languages, but make the progress go from
```

File: frontend_lib/src/component/ProgressBar/ProgressBar.styl [hunk 2]
```diff
@@ -8,6 +8,18 @@
   margin 0 auto
   position relative
 
+// RJ - 2022-02-01 - NOTE
```
Comment (gbonaspetti) [inline frontend_lib/src/component/ProgressBar/ProgressBar.styl:11]:
```suggestion
// INFO - RJ - 2022-02-01 -
```

**Review by gbonaspetti — COMMENTED** [review]
Should we put all [dir=rtl] together? I don't know if it's easy to maintain the code like this.

**Comment by raphj** [comment]
> Should we put all [dir=rtl] together? I don't know if it's easy to maintain the code like this.

I would not. I would keep the direction-specific blocks close to the elements so if one modify the original block, they see the rtl / ltr block just below or above and think of updating them too. But if you prefer to group these direction specific blocks all toghether, it's okay for me. You are more likely to work on this code than me in the future. What do you say @gbonaspetti ?

**Review by gbonaspetti — APPROVED** [review]

**Review by knodelle — APPROVED** [review]

**Pull request merged**